app/domain/authentication/authn/authenticator.rb

```diff
@@ -1,40 +1,54 @@
 # frozen_string_literal: true
 
+require 'command_class'
+
 # default conjur authenticator
-#
 module Authentication
   module Authn
+    Authenticator = CommandClass.new(
+        dependencies: {
+           role_cls:  ::Role,
+           credentials_cls: ::Credentials
+        },
+        inputs:       [:authenticator_input]
+      ) do
```

Comment:
please fix indentation. it should be:
```Authenticator = CommandClass.new(
  dependencies: {
    role_cls:  ::Role,
    credentials_cls: ::Credentials
  },
  inputs:       [:authenticator_input]
) do
```

you can look at authn-oidc for refereance.